pkg/cfn/builder/nodegroup.go

```diff
@@ -251,6 +251,8 @@ func (n *NodeGroupResourceSet) GetAllOutputs(stack cfn.Stack) error {
 }
 
 func newLaunchTemplateData(n *NodeGroupResourceSet) *gfn.AWSEC2LaunchTemplate_LaunchTemplateData {
+	isTInstance := false
```

Comment:
Let's skip the use of this variable at all. If `n.spec.CPUCredits` was set then here we just trust that it has been validated before and set `CpuCredits` accordingly.

Reply:
OK, agree.